src/wallet/walletdb.h

```diff
@@ -58,6 +58,9 @@ enum DBErrors
     DB_NEED_REWRITE
 };
 
+// {value, isHardened}
+typedef pair<int64_t,bool> Component;
```

Comment:
Four bytes is enough. We can use uint32_t follow the function `CPubKey CWallet::GenerateNewKey(uint32_t nChange)`.

https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#extended-keys

Reply:
Agree. Updated